libtock/kernel/syscalls/app_loader_syscalls.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "libtock/tock.h"
+
+// Driver number for system call
+#define DRIVER_NUM_APP_LOADER   0x10001
+
+/*
+ * Command to check if the `app_loader` capsule exists.
+ */
+bool libtock_app_loader_exists(void);
+
+/*
+ * Function to setup the callback from capsule.
+ * This function takes in the function that will be executed
+ * when the callback is triggered.
+ */
+returncode_t libtock_app_loader_setup_subscribe(subscribe_upcall cb, void* userdata);

```

Comment:
These function names need to follow the guide: https://github.com/tock/libtock-c/blob/master/doc/guide.md